scala3/scaladoc.md

```diff
@@ -43,16 +43,18 @@ Furthermore, Scaladoc provides an easy way to configure your [social media links
 
 The following features are currently (May 2021) not stable to be released with scaladoc, however we are happy to hear your feedback. Each feature has its own thread at scala-lang contributors site, where you can share your opinions.
```

Comment:
```suggestion
As of May, 2021, the following features are not currently stable enough to be released with the scaladoc tool, but we are happy to hear your feedback. Each feature has its own thread at the scala-lang contributors site, where you can share your opinions.
```

- Should "scaladoc" be shown as `scaladoc`?
- It would be good to provide a link to the scala-lang contributors site